packages/swc-configs/src/build.ts

```diff
@@ -0,0 +1,55 @@
+import type { Config } from "@swc/core";
+import { resolveOverrides, type ConfigOverride } from "./resolveOverrides.ts";
+
+export const DefaultBuildConfig = {
```

Comment:
Could this be typed :Config?

There is no validation on this object

Reply:
No, otherwise the type would be Config which is not exactly true as most values are not specified by default.

It's more accurate to have an inferred type based on the actual values that are provided by default.

Reply:
satisfies Config or Partial<Config> would be better then

Reply:
Added `satisfities` !